rmw_connext_cpp/CMakeLists.txt

```diff
@@ -27,7 +27,14 @@ ament_target_dependencies(rmw_connext_cpp
   "rosidl_generator_cpp"
   "rosidl_typesupport_connext_cpp"
   "Connext")
+# On Windows this adds the -DRMW_BUILDING_DLL definition.
+# On Unix (GCC or Clang) it hides the symbols with -fvisibility=hidden.
```

Comment:
Rather then adding this comment to every usage of this function the information should be added to the docblock.